unity/Assets/Scripts/BaseFPSAgentController.cs

```diff
@@ -1563,7 +1601,7 @@ public void ProcessControlCommand(dynamic controlCommand)
             lastAction = controlCommand.action;
             lastActionSuccess = false;
             lastPosition = new Vector3(transform.position.x, transform.position.y, transform.position.z);
-            this.actionComplete = false;
+            this.agentState = AgentState.Processing;
```

Comment:
If moved to the `string` argument ProcessControlCommand, maybe remove this on here

Reply:
The string ProcessControlCommand only exists in the AgentManager, so we have to set this here since this is in the actual agent handling the request.